src/clientManager.js

```diff
@@ -20,6 +20,19 @@ ClientManager.prototype.findClient = function(name) {
 	return false;
 };
 
+ClientManager.prototype.writeUser = function(userDict) {
+	var users_path = Helper.HOME + "/users" ;
+	var user_path = users_path + "/" + userDict.user + ".json";
+
+	mkdirp(users_path);
+	fs.writeFileSync(
+		user_path,
+		JSON.stringify(userDict, null, "  ")
+	);
+	// No effect on Windows (and no easy way to do something equivalent)
+	fs.chmodSync(user_path, "0660");
```

Comment:
> […] Big :-1:  […]

Well… We are still in a discussion process, no need to send this PR to death yet, (I totally agree to edit it following the points you and @xPaw raised).

Anyway… will do it… later, nighty night :-)